python/paddle/utils/download.py

```diff
@@ -140,6 +140,21 @@ def _map_path(url, root_dir):
     return osp.join(root_dir, fpath)
 
 
+def _get_unique_endpoints(trainer_endpoints):
+    # Sorting is to avoid different environmental variables for each card
```

Comment:
Move the comment in line 179~181 to here.
This function is mainly used to solve the problem of downloading data from multiple nodes. We select the process with smallest port number of each node to download data, which ensures that only one process of each node downloads data.

Reply:
ok，fix in next pr.